sqrl-math/sqrl-package/README.md

```diff
@@ -0,0 +1,148 @@
+# SQRL Mathematical Functions
+
+The `com.datasqrl.math` package provides a collection of advanced mathematical functions for use within the SQRL framework.
```

Comment:
Let's not duplicate all this content in two places. Since we need to have it here (so it shows up in the repository), my recommendation would be to have this file be the documentation (i.e. a table of the functions with a short demo on how to use one as an example) and have the top-level readme link to this file. The top-level readme should have information on the implementation (i.e. that this is a wrapper around commons math, how it's tested, something short)